tests/serac_input.cpp

```diff
@@ -42,7 +42,7 @@ namespace serac {
 TEST_F(InputTest, vec_1d)
 {
   reader_->parseString("vec = { x = 4.75 }");
-  auto& vec_table = inlet_->addTable("vec");
+  auto& vec_table = inlet_->addStruct("vec");
```

Comment:
oh no! I think I ran my sed script on `src` only! Thanks for catching this.

Reply:
I'm using a new Axom version so I've had to fix it in all of my branches (e.g. #398) to even get it to compile 😆

Reply:
This does keep happening.. should we move unit tests under src next to their units when we move to our more strict integration/unit tests split?